additions-primary-account, DATEV MeinFiskal (#30)
fiskaltrust/portal-manual-doc: doc/handbook-general/company.md

#### Active roles and contractual relationships including API-access

![Active roles and API-access](images/AccountProfile/overview-active-roles-third-parties.png)
Overview - All roles and contractual relationships with fiskaltrust

![Number 1](../images/Numbers/1.png) **Text:** "These roles have been assigned to your company on fiskaltrust.Portal."

![Number 2](../images/Numbers/2.png) **Text:** "Assign further roles to your company:"

![Number 3](../images/Numbers/3.png) **Text:** "Here you can enable connections to 3rd party partners:<br />
**DATEV MeinFiskal**<br />
Your company has not yet been assigned for DATEV MeinFiskal access.  Please change slider to ON to enable access on the next page."

![Number 4](../images/Numbers/4.png) **Text:** "API access

> Your cash register`s fiskaltrust.Helipad online-functions will need an account ID and an access token to work automatically.<br />

Reviewer: I think this one sentence could need some clarification.

![Number 7](../images/Numbers/7.png) The button `[Reset]` can be used to regenerate the account ID and the AccessToken.

#### Company employees

Add new employees to your company and manage their permissions.
![Company- Employees](images/AccountProfile/Employee-with-primary.png)
Company - Employees

![Number 1](../images/Numbers/1.png) By default, the primary contact of the company is the user who registered the company as an employee. The highlighted record is the one with the primary role.

![Number 2](../images/Numbers/2.png) By clicking on the arrow next to the First name, the sliders for changing the authorizations are displayed.

The different authorizations are described in chapter "[Employees of the company authorization administrations](#Employees-of-the-company-authorization-administration)".

![Number 3](../images/Numbers/3.png) In this case, the primary contact of the company is displayed. This user always has full rights. These authorizations cannot be reset either. Therefore the sliders are grey and cannot be changed.

> If no employee has administrative rights, you lose the ability to access the fiskaltrust.Portal. In this situation fiskaltrust.Support has to be contacted.

Reviewer 2: ```suggestion
If no employee has administrative rights, you lose the ability to access the fiskaltrust.Portal. In this situation fiskaltrust.CustomerSuccess has to be contacted.
```

Reviewer: Thanks, @AxelKutschera, though I think we'll need to revisit the terminology once more, as _fiskaltrust.Support_ is currently only in one document mentioned and _fiskaltrust.CustomerSuccess_ in _none_. Tagging monsieur @fabry-training as well 😇

Reviewer 3: I liked to use Support. Because that's really a task done by the subdivision Support of the CustomerSuccess Team, isn't it?

Reviewer 4: thank you, IMHO that is a question of terminology, another PBI - we follow that in another task

> If no employee has the administrative rights, you lose the possibility to access thee fiskaltrust portal. In this situation the [fiskaltrust.support](mailto:support@fiskaltrust.de) has to be contacted.

Reviewer 3: NEVER use eMail-Addresses for a specific market in general parts.

Reviewer 4: ```suggestion
If no employee has the administrative rights, you lose the possibility to access thee fiskaltrust portal. In this situation the fiskaltrust.Support has to be contacted.
```

Reviewer 4: thx

Reviewer 3: and please fiskaltrust.Support

> If no employee has the administrative rights, you lose the possibility to access thee fiskaltrust portal. In this situation the [fiskaltrust.support](mailto:support@fiskaltrust.de) has to be contacted.

Reviewer: ```suggestion
If no employee has administrative rights, you lose the ability to access the fiskaltrust.Portal. In this situation fiskaltrust.Support has to be contacted.
```
